spec.md

```diff
@@ -1137,26 +1137,27 @@ message NodeUnpublishVolumeRequest {
   // This is a REQUIRED field.
   string target_path = 3;
 
-  // End user credentials used to authenticate/authorize node
+  // Credentials used by Node plugin to authenticate/authorize node
   // unpublish request.
   // This field contains credential data, for example username and
   // password. Each key must consist of alphanumeric characters, '-',
   // '_' or '.'. Each value MUST contain a valid string. An SP MAY
   // choose to accept binary (non-string) data by using a binary-to-text
   // encoding scheme, like base64. An SP SHALL advertise the
   // requirements for credentials in documentation. COs SHALL permit
-  // users to pass through the required credentials. This information is
+  // passing through the required credentials. This information is
   // sensitive and MUST be treated as such (not logged, etc.) by the CO.
   // This field is OPTIONAL.
-  map<string, string> user_credentials = 4;
+  map<string, string> node_unpublish_credentials = 4;
 }
 
 message NodeUnpublishVolumeResponse {}
 ```
 
 ##### NodeUnpublishVolume Errors
 
-If the plugin is unable to complete the NodeUnpublishVolume call successfully, it MUST return a non-ok gRPC code in the gRPC status.
+If the plugin is unable to
```

Comment:
please undo this seemingly unintentional change

Reply:
thank you for catching it, fixed.